### PR TITLE
Add sharedController property to AccountController & SessionController

### PR DESCRIPTION
**Pull request by stephanheilner**

### CODE REVIEW

File: LDSAnnotations/SessionController.swift [hunk 1]
```diff
@@ -24,16 +24,14 @@ import Foundation
 
 public class SessionController {
```
Comment (stephanheilner) [inline LDSAnnotations/SessionController.swift]: I wonder if this would be better named `SessionCache` now, or something like that?

**Comment by nickmshelley** [comment]
What's happening with this? Are you planning on changing anything or is this ready for review? I can't remember what we decided on this.

**Comment by stephanheilner** [comment]
I have a few other changes I need to make. I think we stick with sharedControllers for the time-being. The changes in the current version of the PR aren't actually making any singletons, just using static vars and class functions, making it act like a singleton would. Stay tuned for another commit...

**Comment by nickmshelley** [comment]
Sounds good.

**Comment by stephanheilner** [comment]
If @hiltonc feels strongly about not including the sharedController static lets to the pod, then I can add the extra extensions to the demo project and GL to have the sharedController

**Comment by hiltonc** [comment]
:+1: I think this is great. Thank you for not using statics other than the shared instances. I agree that nearly, if not all, library users will use that.

**Comment by stephanheilner** [comment]
Thanks @hiltonc!

**Pull request merged**